HTMLSerializer.js

```diff
@@ -122,8 +160,8 @@ var HTMLSerializer = class {
     var win = element.ownerDocument.defaultView;
     var style = win.getComputedStyle(element, null).cssText;
     var windowDepth = this.windowDepth(win);
-    style = style.replace(/"/g, this.escapedQuote(windowDepth+1));
-    var quote = this.escapedQuote(windowDepth);
+    style = style.replace(/"/g, this.escapedCharacter('"', windowDepth+1));
+    var quote = this.escapedCharacter('"', windowDepth);
```

Comment:
There are 4 places where the same pattern of quote+name+value+quote is used. WDYT about unifying them behind two functions? Maybe processSimpleAttribute and a new function "processHoleAttribute(win, name)" that returns the index of the hole?